app/components/Form/SelectProjectStateWidget.tsx

```diff
@@ -0,0 +1,91 @@
+import { WidgetProps } from "@rjsf/core";
+import { useMemo } from "react";
+import { graphql, useFragment } from "react-relay";
+import Dropdown from "@button-inc/bcgov-theme/Dropdown";
+
+const SelectProjectStatus: React.FunctionComponent<WidgetProps> = (props) => {
+  const {
+    id,
+    onChange,
+    placeholder,
+    label,
+    required,
+    uiSchema,
+    value,
+    formContext,
+  } = props;
+
+  const { allProjectStates, allFundingStreamRfpProjectStates } = useFragment(
+    graphql`
+      fragment SelectProjectStateWidget_query on Query {
+        allFundingStreamRfpProjectStates {
+          edges {
+            node {
+              rowId
+              fundingStreamRfpId
+              projectStateId
```

Comment:
Instead of fetching the `projectStateId`, and iterating on `allProjectStates` below, you could follow the projectByProjectStateId relation:
```suggestion
              projectStateByProjectStateId {
                rowId
                name
              }
```